lib/gui/app.js

```diff
@@ -346,6 +346,17 @@ app.controller('StateController', function ($rootScope, $scope) {
   this.currentName = null
 })
 
+// Handle [Cmd] + [,] shortcut to open the settings
```

Comment:
Given that the ` [Cmd] + [,] shortcut` is encapsulated inside `lib/gui/menu.js`, perhaps this comment should just mention the `preferences:toggle` action? \*shrug\*

Reply:
Yeah, I agree. This part of the code doesn't care what the shortcut is.

Reply:
Ah yes, missed this comment before. Updated now!